src/RoleObjectFactory.ts

```diff
@@ -53,30 +55,42 @@ import TreeItemData from './RoleObjects/TreeItemData';
 import ToolBarData from './RoleObjects/ToolBarData';
 import UnorderedListData from './RoleObjects/UnorderedListData';
 
+type EventHandler = (evt) => void;
```

Comment:
Missing type for `evt`
```suggestion
type EventHandler = (evt: Event) => void;
```